src/index.js

```diff
@@ -1,3 +1,41 @@
+<<<<<<< HEAD
+import $ from 'jquery';
+import Deck from '../src/Deck.js'
+var data
+fetch("https://fe-apps.herokuapp.com/api/v1/gametime/1903/jeopardy/data")
+.then(response => response.json())
+.then(fetchData => data = fetchData.data)
+
+$(document).ready(() => {
+  setTimeout(function() {
```

Comment:
This is a kind of hacky way to wait for your data to come in. I’d recommend doing a bit more logic inside of that second `.then()` of your fetch request, where you know the data has come back and it’s safe to kick off your new game.